src/smart-escrow/SmartEscrow.sol

```diff
@@ -242,12 +259,22 @@ contract SmartEscrow is AccessControlDefaultAdminRules {
     /// @notice Returns the amount vested as a function of time.
     /// @param _timestamp The timestamp to at which to get the vested amount
     function _vestingSchedule(uint256 _timestamp) internal view returns (uint256) {
-        if (_timestamp < start) {
+        if (_timestamp < cliffStart) {
             return 0;
         } else if (_timestamp > end) {
             return OP_TOKEN.balanceOf(address(this)) + released;
         } else {
-            return initialTokens + ((_timestamp - start) / vestingPeriod) * vestingEventTokens;
+            return _cliffRelease(_timestamp) + ((_timestamp - start) / vestingPeriod) * vestingEventTokens;
+        }
+    }
+
+    /// @notice Returns the cliff amount if past the cliffStart time
+    /// @param _timestamp The timestamp to at which to get cliff release status 
+    function _cliffRelease(uint256 _timestamp) internal view returns (uint256) {
+        if (_timestamp < cliffStart) {
+            return 0;
+        } else {
+            return initialTokens;
```

Comment:
Wondering why we need this function -- not sure why we can't simply return `initialTokens` above, since this should always return `initialTokens` based on how it's used here. The condition `_timestamp < cliffStart` is always expected to be true when this function is called from `_vestingSchedule`

Reply:
Good catch! I implemented this before we added the strict time gating with start < cliffStart < end. Will remove and cleanup.